composer/algorithms/low_precision_layernorm/low_precision_layernorm.py

```diff
@@ -72,18 +70,17 @@ def to_LPLayerNorm(layer: torch.nn.Module, module_index: int) -> LPLayerNorm:
 
 def to_FusedLayerNorm(layer: torch.nn.Module, module_index: int) -> APEXFusedLayerNorm:
     """Defines a replacement policy from a `torch.nn.LayerNorm` to a `apex.normalization.fused_layer_norm`"""
-    assert isinstance(layer,
-                      torch.nn.LayerNorm), 'The replacement policy will look for all instances of torch.nn.LayerNorm'
+    assert isinstance(layer, torch.nn.LayerNorm), 'Replacement policy only replaces torch.nn.LayerNorm'
```

Comment:
technically this should be a TypeError since [asserts are more for detecting bugs / documenting program logic, not raising user-visible exceptions](https://stefan.sofa-rockers.org/2018/04/16/assertions-and-exceptions/)